modules/installation/pages/general-requirements.adoc

```diff
@@ -42,15 +42,20 @@ Depending on your organization's setup, you might also need to activate your sub
 
 
 [[install.media]]
-== Obtaining Installation Media
+== Obtaining {slsa} Unified Installer
```

Comment:
is the {slsa} really necessary here? Also, it should probably be "Obtain"

Reply:
Maybe {sle}, maybe not at all. Yes.